src/components/SettingsToggle.js

```diff
@@ -1,17 +1,20 @@
 import PropTypes from 'prop-types';
-import React, { useEffect, useState } from 'react';
+import React, { useContext, useEffect, useState } from 'react';
 import { ActivityIndicator, StyleSheet } from 'react-native';
 import { ListItem } from 'react-native-elements';
 
-import { colors, consts, device, normalize } from '../config';
+import { colors, consts, device, normalize, texts } from '../config';
+import { NetworkContext } from '../NetworkProvider';
+import { serverConnectionAlert } from '../pushNotifications';
 
 import { Switch } from './Switch';
 import { BoldText, RegularText } from './Text';
 import { Touchable } from './Touchable';
 import { WrapperRow } from './Wrapper';
 
 // TODO: snack bar / toast als nutzerinfo
-export const SettingsToggle = ({ item }) => {
+export const SettingsToggle = ({ isNeedConnection = true, item }) => {
```

Comment:
is need connection? what about needs connection? i think this would grammatically correct.